devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy

```diff
@@ -362,7 +364,7 @@ def handleTemperature(descMap) {
 			// just mask it.
 			if (!cleared &&
 					((lastAlarm == "freeze" && map.value > FREEZE_ALARM_TEMP) ||
-					 (lastAlarm == "heat" && map.value < HEAT_ALARM_TEMP))) {
+					(lastAlarm == "heat" && map.value < HEAT_ALARM_TEMP))) {
```

Comment:
I'd leave the space there so it is more clear that `(lastAlarm == "heat" && map.value < HEAT_ALARM_TEMP)` is still part of the overall group.